Update backprop.py with better markdown

jvishnuvardhan opened the pull request.

jvishnuvardhan committed:
Update backprop.py

dynamicwebpaige reviewed (dismissed):
LGTM! Thanks for the fixes. 🙂

rthadur commented:
@yifeif can you please help merge this PR

jvishnuvardhan committed:
Updated batch_jacobian example

Updated batch_jacobian example that was throwing a warning. Github [gist](https://colab.sandbox.google.com/gist/jvishnuvardhan/b15001a46f9acc172b6a77dd2d2620c4/untitled172.ipynb) attached for your reference. Thanks!

rthadur commented:
Closing this PR as this not against `master`, please open a new PR against `master` 
CC @mihaimaruseac
